crates/worker/src/partition/services/non_deterministic/mod.rs

```diff
@@ -65,14 +61,14 @@ impl ServiceInvoker {
     pub(crate) async fn invoke(
         &mut self,
         full_invocation_id: FullInvocationId,
-        method: &str,
+        _method: &str,
```

Comment:
What's the idea to keep the non-deterministic services around? Are there any plans to add other non-deterministic services in the future? Removing it, would allow us to also clean up the partition processor's command interpreter a bit more.

Reply:
last time we talked about this, I thought we pretty much agreed on keeping that in case we might need it. IDK if we wanna use this infra for durable promises... If we wanna remove those, i'm ok with doing that, but then i'll do this in another PR.